client/src/pages/board/components/dashboardForms/OpenCardDialog.js

```diff
@@ -0,0 +1,85 @@
+import {Button, colors} from '@material-ui/core';
+import React, {useState} from 'react';
+import CardDialog from './CardDialog';
+
+// TODO(FS): Create color labels for users, where they
+//           can edit/define new colors and their associated
+//           labels.
+const sampleCard = {
+  title: 'Sample Card',
+  columnName: 'Col',
+  desc:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas dictum dolor at eleifend aliquam. Cras condimentum erat eget purus fermentum faucibus. Pellentesque habitant morbi tristique senectus et netus et malesuada.',
+  checklist: [
+    {timestamp: 123123, checked: true, text: 'Do A,B and C'},
+    {
+      timestamp: 123,
+      checked: false,
+      text:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas dictum dolor at eleifend aliquam. Cras condimentum erat eget purus fermentum faucibus. Pellentesque habitant morbi tristique senectus et netus et malesuada.',
+    },
+  ],
+  deadline: Date.now(),
+  tags: ['math', 'exam', 'important'],
+  color: colors.amber[500],
+  comments: [
+    // NOTE: The timestamp is intended to
+    //      serve as a unique id to distinguish comments
+    //      on frontend. As far as the backend is concerned
+    //      this is just part of the data to save.
+    {
+      timestamp: 0,
+      comment:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ornare quam ac condimentum accumsan. Nulla eu efficitur dui, vitae gravida felis. In vitae turpis at nunc tincidunt vestibulum. Nulla facilisi. Nunc nisi est, efficitur in lobortis at, dapibus ac diam. Nunc a elit volutpat enim sodales porttitor at id lacus. Cras non tempor justo.',
+    },
+    {
+      timestamp: 1,
+      comment:
+        'Donec eu semper lacus, vitae vehicula nisl. Nullam sed mi vulputate, laoreet tellus id, porta neque. Nullam semper placerat nisi, sit amet gravida est gravida vel. Ut ut purus pulvinar velit ultrices pellentesque vel sit amet nisi.',
+    },
+  ],
+  attachements: null,
+};
```

Comment:
Is this sampleCard necessary? Judging by the lorem ipsum dolors, it looks like it was made mostly for testing but OpenEditCardDialog uses its values as props.

Reply:
Nope! Actually the whole component is unnecessary. Forgot to delete that file, thanks for catching it.